modules/moordyn/src/MoorDyn_Point.f90

```diff
@@ -379,14 +373,15 @@ SUBROUTINE Point_RemoveLine(Point, lineID, TopOfLine, rEnd, rdEnd)
       REAL(DbKi),       INTENT(INOUT)    :: rdEnd(3)
       
       Integer(IntKi)    :: l,m,J
+      Integer(IntKi)    :: found = 0
```

Comment:
In Fortran, initializing a variable in the type declaration statement automatically gives it a "SAVE" attribute, which means it retains its value each time it is called (i.e., it's set to 0 only the first time the routine is called). Since this is called from an `UpdateStates` routine, I'd be surprised if this is how it was intended to be used.... and if that _was_ the intention, it wouldn't work properly in a restart situation. 

I'd recommend either 

1. rewriting like this:
```
Integer(IntKi) :: found

found = 0
``` 
or 

2. putting this variable in a `MiscVar` type, if you do intend for it to have the `SAVE` attribute.

Reply:
Also, using a `LOGICAL` here might make it easier to read, unless you have plans for checking something other than found vs not-found .

Reply:
@andrew-platt if you have a PR or something merging soon can you add this `found = 0` and the `logical` fix there? Otherwise I can open a PR. It's intent is a boolean flag that prints an error message (I probably should've made it throw a warning)

Reply:
I'll include this into a minor PR that I'll post tomorrow.

Reply:
Thanks!

Reply:
See #2400